src/Microsoft.ML.Ensemble/OutputCombiners/BaseStacking.cs

```diff
@@ -185,7 +185,7 @@ public void Train(List<FeatureSubsetModel<IPredictorProducing<TOutput>>> models,
                 bldr.AddColumn("Features", NumberType.Float, features);
 
                 var view = bldr.GetDataView();
-                var rmd = RoleMappedData.Create(view, ColumnRole.Label.Bind("Label"), ColumnRole.Feature.Bind("Features"));
+                var rmd = new RoleMappedData(view, "Label", "Features");
```

Comment:
>"Label", "Features" [](start = 51, length = 19)

can we replace it with DefaultColumnNames.Label, DefaultColumnNames.Features
same for line 184 and line 185

Reply:
Ah good one. Also SmacSweeper...

---
In reply to: [200460321](https://github.com/dotnet/machinelearning/pull/496#discussion_r200460321) [](ancestors = 200460321)